pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -218,6 +219,7 @@ buildPythonApplication rec {
     license = licenses.gpl3Only;
     changelog = "https://sw.kovidgoyal.net/kitty/changelog/";
     platforms = platforms.darwin ++ platforms.linux;
-    maintainers = with maintainers; [ tex rvolosatovs Luflosi ];
+    broken = (stdenv.isDarwin && stdenv.isx86_64);
```

Comment:
```suggestion
    broken = stdenv.isDarwin && stdenv.isx86_64;
```

Reply:
I'll try and remember this for the next version bump if I beat the bot.

Reply:
just remember it for the next time, just some styling